Build errors

Hello!
I trying to build plugin, but getting a lot of errors:

> make: go: Permission denied
> make: go: Permission denied
> ./build/bin/manifest apply
> mkdir -p server/dist;
> cd server && env CGO_ENABLED=0 GOOS=linux GOARCH=amd64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-linux-amd64;
> cd server && env CGO_ENABLED=0 GOOS=linux GOARCH=arm64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-linux-arm64;
> cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-darwin-amd64;
> cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-darwin-arm64;
> cd server && env CGO_ENABLED=0 GOOS=freebsd GOARCH=amd64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-freebsd-amd64;
> cd server && env CGO_ENABLED=0 GOOS=openbsd GOARCH=amd64 /usr/local/bin/go/bin/go build  -ldflags '-X "main.buildHash=eb6cec4f69a4b2b9dc11d43d45c9b14ebbcb3d6f" -X "main.isDebug=" -X "main.rudderWriteKey=" -X "main.rudderDataplaneURL="' -trimpath -o dist/plugin-openbsd-amd64;
> cd webapp && node skip_integrity_check.js
> cd webapp && /usr/local/bin/npm install
> npm WARN EBADENGINE Unsupported engine {
> npm WARN EBADENGINE   package: 'eslint-import-resolver-webpack@0.13.1',
> npm WARN EBADENGINE   required: {
> npm WARN EBADENGINE     node: '^16 || ^15 || ^14 || ^13 || ^12 || ^11 || ^10 || ^9 || ^8 || ^7 || ^6'
> npm WARN EBADENGINE   },
> npm WARN EBADENGINE   current: { node: 'v18.13.0', npm: '9.3.0' }
> npm WARN EBADENGINE }
> npm WARN skipping integrity check for git dependency ssh://git@github.com/mattermost/react-bootstrap.git 
> npm WARN skipping integrity check for git dependency ssh://git@github.com/mattermost/marked.git 
> npm WARN skipping integrity check for git dependency ssh://git@github.com/mattermost/dynamic-virtualized-list.git 
> npm WARN deprecated @types/emoji-regex@9.2.0: This is a stub types definition. emoji-regex provides its own type definitions, so you do not need this installed.
> npm WARN deprecated natives@1.1.6: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
> npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
> npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
> npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
> npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
> npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
> npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
> npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
> npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
> npm ERR! code 1
> npm ERR! git dep preparation failed
> npm ERR! command /usr/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/build/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
> npm ERR! npm WARN using --force Recommended protections disabled.
> npm ERR! npm WARN deprecated natives@1.1.6: This module relies on Node.js's internals and will break at some point. Do not use it, and update to graceful-fs@4.x.
> npm ERR! npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
> npm ERR! npm WARN deprecated @storybook/addon-knobs@6.3.0: deprecating knobs in favor of @storybook/addon-controls
> npm ERR! npm ERR! code 1
> npm ERR! npm ERR! git dep preparation failed
> npm ERR! npm ERR! command /usr/bin/node /usr/local/lib/node_modules/npm/bin/npm-cli.js install --force --cache=/home/build/.npm --prefer-offline=false --prefer-online=false --offline=false --no-progress --no-save --no-audit --include=dev --include=peer --include=optional --no-package-lock-only --no-dry-run
> npm ERR! npm ERR! > mmjstool@1.0.0 prepare
> npm ERR! npm ERR! > webpack
> npm ERR! npm ERR! npm WARN using --force Recommended protections disabled.
> npm ERR! npm ERR! npm WARN skipping integrity check for git dependency ssh://git@github.com/mattermost/eslint-config-mattermost.git 
> npm ERR! npm ERR! npm WARN deprecated set-value@0.4.3: Critical bug fixed in v3.0.1, please upgrade to the latest version.
> npm ERR! npm ERR! npm WARN deprecated set-value@2.0.0: Critical bug fixed in v3.0.1, please upgrade to the latest version.
> npm ERR! npm ERR! npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
> npm ERR! npm ERR! npm WARN deprecated har-validator@5.1.3: this library is no longer supported
> npm ERR! npm ERR! npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
> npm ERR! npm ERR! npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
> npm ERR! npm ERR! npm WARN deprecated chokidar@2.0.4: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
> npm ERR! npm ERR! npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
> npm ERR! npm ERR! npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
> npm ERR! npm ERR! npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
> npm ERR! npm ERR! npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
> npm ERR! npm ERR! npm WARN deprecated babel-eslint@10.0.1: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
> npm ERR! npm ERR! npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
> npm ERR! npm ERR! npm WARN deprecated mkdirp@0.5.1: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
> npm ERR! npm ERR! npm WARN deprecated request@2.88.0: request has been deprecated, see https://github.com/request/request/issues/3142
> npm ERR! npm ERR! npm WARN deprecated buffer@4.9.1: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
> npm ERR! npm ERR! npm WARN deprecated core-js@2.6.2: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
> npm ERR! npm ERR! node:internal/crypto/hash:71
> npm ERR! npm ERR!   this[kHandle] = new _Hash(algorithm, xofLen);
> npm ERR! npm ERR!                   ^
> npm ERR! npm ERR! 
> npm ERR! npm ERR! Error: error:0308010C:digital envelope routines::unsupported
> npm ERR! npm ERR!     at new Hash (node:internal/crypto/hash:71:19)
> npm ERR! npm ERR!     at Object.createHash (node:crypto:133:10)
> npm ERR! npm ERR!     at module.exports (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/webpack/lib/util/createHash.js:90:53)
> npm ERR! npm ERR!     at NormalModule._initBuildHash (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/webpack/lib/NormalModule.js:401:16)
> npm ERR! npm ERR!     at handleParseError (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/webpack/lib/NormalModule.js:449:10)
> npm ERR! npm ERR!     at /home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/webpack/lib/NormalModule.js:481:5
> npm ERR! npm ERR!     at /home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/webpack/lib/NormalModule.js:342:12
> npm ERR! npm ERR!     at /home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/loader-runner/lib/LoaderRunner.js:373:3
> npm ERR! npm ERR!     at iterateNormalLoaders (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
> npm ERR! npm ERR!     at Array.<anonymous> (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
> npm ERR! npm ERR!     at Storage.finished (/home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:43:16)
> npm ERR! npm ERR!     at /home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:79:9
> npm ERR! npm ERR!     at /home/build/.npm/_cacache/tmp/git-clone8NSO0M/node_modules/graceful-fs/graceful-fs.js:90:16
> npm ERR! npm ERR!     at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
> npm ERR! npm ERR!   opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
> npm ERR! npm ERR!   library: 'digital envelope routines',
> npm ERR! npm ERR!   reason: 'unsupported',
> npm ERR! npm ERR!   code: 'ERR_OSSL_EVP_UNSUPPORTED'
> npm ERR! npm ERR! }
> npm ERR! npm ERR! 
> npm ERR! npm ERR! Node.js v18.13.0
> npm ERR! npm ERR! npm ERR! code 1
> npm ERR! npm ERR! npm ERR! path /home/build/.npm/_cacache/tmp/git-clone8NSO0M
> npm ERR! npm ERR! npm ERR! command failed
> npm ERR! npm ERR! npm ERR! command sh -c webpack
> npm ERR! npm ERR! 
> npm ERR! npm ERR! npm ERR! A complete log of this run can be found in:
> npm ERR! npm ERR! npm ERR!     /home/build/.npm/_logs/2023-01-17T21_09_11_147Z-debug-0.log
> npm ERR! 
> npm ERR! npm ERR! A complete log of this run can be found in:
> npm ERR! npm ERR!     /home/build/.npm/_logs/2023-01-17T21_08_39_214Z-debug-0.log
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/build/.npm/_logs/2023-01-17T21_07_56_021Z-debug-0.log
> make: *** [Makefile:132: webapp/node_modules] Error 1

Ubuntu Jammy 22.04
go version go1.19.5 linux/amd64
Node.js v18.13.0
npm@9.3.0

--- a/README.md
+++ b/README.md
@@ -22,6 +22,13 @@ This plugin demands some network configuration changes to allow audio/video comm
 
 ## Development
 
+> **_Note_**
+>
+> Building the plugin requires the following:
+> - Golang: version >= **1.18**
+> - NodeJS: version **16.x**
+> - NPM: version **8.x**
+
 Use ```make dist``` to build this plugin.
 
 Use `make deploy` to deploy the plugin to your local server.

